### PR TITLE
Add ability to attach a replicated release to an unreleased EC binary

### PR DESCRIPTION
**Pull request by marccampbell**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -359,3 +359,49 @@ test-lam-e2e: pkg/goods/bins/local-artifact-mirror
 bin/installer:
 	@mkdir -p bin
 	go build -o bin/installer ./cmd/installer
+
+# make test-embed channel=Unstable app=slackernews
+.PHONY: test-embed
+test-emded: export OS=linux
+test-embed: export ARCH=amd64
```
Comment (sgalsaleh) [inline Makefile:366]: This means we can't use this in the dev env no?

File: Makefile [hunk 2]
```diff
@@ -359,3 +359,49 @@ test-lam-e2e: pkg/goods/bins/local-artifact-mirror
...
+test-embed: static embedded-cluster
+	@echo "Cleaning up previous release directory..."
```
Comment (sgalsaleh) [inline Makefile:368]: Feels like we should move this to dagger or a script. Seems overloaded to have all of this logic in the makefile

File: Makefile [hunk 3]
```diff
@@ -359,3 +359,49 @@ test-lam-e2e: pkg/goods/bins/local-artifact-mirror
...
+	rm -rf ./hack/release
```
Comment (sgalsaleh) [inline Makefile:369]: We use the "build" directory in other places for gitignored artifacts

File: Makefile [hunk 4]
```diff
@@ -359,3 +359,49 @@ test-lam-e2e: pkg/goods/bins/local-artifact-mirror
 bin/installer:
 	@mkdir -p bin
 	go build -o bin/installer ./cmd/installer
+
+# make test-embed channel=Unstable app=slackernews
+.PHONY: test-embed
```
Comment (sgalsaleh) [inline Makefile:364]: Why not use the embedded-cluster-release-builder binary we already have? We could replace it with this one I don't mind.

File: Makefile (hunk 1 above)
Comment (marccampbell) [reply Makefile:366]: it won't affect, this exports them only in this 1 new make target

File: Makefile (hunk 3 above)
Comment (marccampbell) [reply Makefile:369]: ok, but this is not a build artifact, the build artifact is in the build dir   this is added to gitignore

File: Makefile (hunk 4 above)
Comment (marccampbell) [reply Makefile:364]: this is a little more specific to the workflow i am trying to crfeate

**Review by sgalsaleh — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (sgalsaleh) [reply Makefile:366]: I understand. I'm saying we won't be able to benefit from it. Which might be fine

**Pull request merged**